components/support/sync-telemetry/src/main/java/mozilla/components/support/sync/telemetry/SyncTelemetry.kt

```diff
@@ -284,4 +287,42 @@ object SyncTelemetry {
         val message = reason.message ?: "Unexpected error: ${reason.code}"
         metric.set(message.take(MAX_FAILURE_REASON_LENGTH))
     }
+
+    fun processFxaTelemetry(jsonStr: String) {
```

Comment:
Maybe you could link to wherever this schema is defined in the app-services repo? It'll be nice to have that handy here.